Tests/SwiftDocCTests/Diagnostics/DiagnosticTests.swift

```diff
@@ -192,11 +192,8 @@ class DiagnosticTests: XCTestCase {
         let engine1 = DiagnosticEngine()
 
         let _ = DocumentationNode.contentFrom(documentedSymbol: symbolWithKnownDirective, documentationExtension: nil, engine: engine1)
-
-        // count should 1 for the known directive '@Image'
+        XCTAssertEqual(engine.problems.count, 0)
```

Comment:
I think it would be better to adjust the input to the test so we can still confirm that unsupported directives throw an error.


Now that `@Image` is supported – maybe we could replace the content on line `188` with a `@TitleHeading` directive instead?